ccache_conf.in

```diff
@@ -1,4 +1,4 @@
 cache_dir = "@MINGW_INSTALL_PREFIX@/ccache"
-max_size = 500M
+max_size = 1G
```

Comment:
Why? 500M big enough for packages building as I dont want the packages' caches grow over time in CI